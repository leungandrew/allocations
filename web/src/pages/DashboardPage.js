import React from 'react';
import ProjectsList from '../components/ProjectsList';

const DashboardPage = () => {
  return (
    <div id='container'>
      <ProjectsList />
    </div>
  );
}

export default DashboardPage;
