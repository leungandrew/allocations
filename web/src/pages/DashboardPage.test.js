import React from 'react'
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import DashboardPage from './DashboardPage';

describe('DashboardPage', () => {
  it('renders ProjectLists', () => {
    const tree = renderer.create(<DashboardPage/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
});
