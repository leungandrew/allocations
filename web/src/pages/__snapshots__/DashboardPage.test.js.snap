// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardPage renders ProjectLists 1`] = `
<div
  id="container"
>
  <div>
    <li>
      Project A
    </li>
    <li>
      Project B
    </li>
  </div>
</div>
`;
